### PR TITLE
Add support for configuration files

### LINKED ISSUE #5: Use config file instead of long command line arguments
An argument like `--config=config.conf` would be great, the commands become really long with all the settings.

### PR DESCRIPTION
**Pull request by Androbin**
Fixes #5
Deconflicts #7

### CODE REVIEW

File: servlet.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 from tornado import httpclient
 from tornado import gen
 from tornado import escape
+import configparser
```
Comment (sushain97) [inline servlet.py]: These imports are a complete mess. Would you mind breaking them into two blocks, one standard libraries and another external libraries and sorting each alphabetically?

File: servlet.py [hunk 2]
```diff
@@ -1345,18 +1380,36 @@ def sanity_check():
...
+            apply_config(args, apySection)
+
     if args.daemon:
         # regular content logs are output stderr
         # python messages are mostly output to stdout
         # hence swapping the filenames?
         sys.stderr = open(os.path.join(args.log_path, 'apertium-apy.log'), 'a+')
         sys.stdout = open(os.path.join(args.log_path, 'apertium-apy.err'), 'a+')
 
-    logging.getLogger().setLevel(logging.INFO)
```
Comment (sushain97) [inline servlet.py]: Why did these lines get deleted?

Comment (Androbin) [reply servlet.py]: There were just moved. See lines 1386/1387.

File: servlet.py (hunk 1 above)
Comment (Androbin) [reply servlet.py]: Sorting? Yeah, sure. Standard vs. External? Not sure how to tell.

Comment (sushain97) [reply servlet.py]: I think only tornado is external.

Comment (sushain97) [reply servlet.py]: eh, 3rd-party library is probably a better term

Comment (sushain97) [reply servlet.py]: oh, typing and tornado are now. the others can all just go in one sorted block

File: servlet.py [hunk 3]
```diff
@@ -1304,6 +1311,40 @@ def sanity_check():
...
+                    if apySection[name] == "False":
+                        res = False
+                    elif apySection[name] == "True":
+                        res = True
+                    else:
+                        res = bool(apySection[name])
+                else:
+                    res = fn(apySection[name])
+            except ValueError:
+                pass
```
Comment (sushain97) [inline servlet.py]: Should we print some sort of warning here?

**Comment by sushain97** [comment]
Waiting for the build to finish on this...

File: servlet.py (hunk 3 above)
Comment (Androbin) [reply servlet.py]: How about `print("Unable to cast " + apySection[name] + " to expected type")` ?

**Comment by sushain97** [comment]
flake might still be unhappy with you :)

**Comment by Androbin** [comment]
Okay, should be fixed now.

**Comment by Androbin** [comment]
Okay, should be fixed now.

**Pull request merged**